### ISSUE TITLE
mounika nfc bug1

### ISSUE THREAD

**Issue by mounikasadineni01** [ALPREDDYY/Health-E-Connect#4]
'alt' attributes for image elements are missing on the homepage. This is a significant accessibility concern

**Comment by mounikasadineni01** [comment]
fixed the issue with pull request #6
